CyberpunkRED_Gerdofal/sheet.json

```diff
@@ -0,0 +1,8 @@
+{
+	"html": "CyberpunkRED.html",
+	"css": "CyberpunkRED.css",
+	"authors": "Gerdofal",
+	"roll20userid": "380243",
+	"preview": "tab1_sheet.png",
```

Comment:
The preview needs to exactly match the name of the file you are uploading. That includes capitalization.  Your preview image is named "tab1_sheet.PNG" not "tab1_sheet.png"

Reply:
Fixed